src/normalize/glossary.ts

```diff
@@ -174,6 +167,13 @@ export default function normalizeGlossary(editor: Editor, entry: NodeEntry): boo
         }
     }
 
+    // We look for Term in DefinitionTerm, unwrap it if found
+    if (Term.isTerm(node) || Title.isTitle(node) || Paragraph.isParagraph(node)) {
+        if (Editor.above(editor, { at: path, match: DefinitionTerm.isDefinitionTerm }) != null) {
+            Transforms.unwrapNodes(editor, { at: path })
+        }
+    }
```

Comment:
This should be grouped with normalizations for `Term` in `src/normalize/text.ts`.